BookingsAPI/Bookings.DAL/Commands/AnonymiseHearingsCommand.cs

```diff
@@ -176,6 +178,15 @@ public async Task Handle(AnonymiseHearingsCommand command)
 			"CLOSE participant_cursor; " +
 			"DEALLOCATE participant_cursor; ";
 			command.RecordsUpdated = await _context.Database.ExecuteSqlRawAsync(query);
-        }
+
+			var updatedDate = DateTime.UtcNow;
+			
+			var updateQuery = @$"IF EXISTS(select * from JobHistory)
+								Update JobHistory Set LastRunDate= '{updatedDate}'
+							ELSE
+							  Insert into JobHistory values (NEWID(), '{updatedDate}')";
+			await _context.Database.ExecuteSqlRawAsync(updateQuery);
```

Comment:
Hi Sunil, We should only add a record in the jobhistory table if there is a successful run. Can you add that check please?

Reply:
Hi Ru, it adds only one record. That's why there's an exists condition :)